external/CapnProto.cmake

```diff
@@ -114,6 +122,7 @@ foreach (INCLUDE_DIR ${CAPNP_INCLUDE_DIRS})
   install(DIRECTORY ${INCLUDE_DIR}/capnp
```

Comment:
Yes they are different. The externals are configured to install into <build_dir>/ThirdParty/Install (or something like that, it's an ExternalProject convention). This is great for most cases where the installed artifacts are only needed for the build of nupic.core. But for capnp we want to also include the capnp install artifacts in the installation of nupic.core and that is what is specified in src/CMakeLists.txt (which is responsible for what gets installed to the user-specified install location).